src/adapters/github.js

```diff
@@ -170,7 +170,8 @@ class GitHub extends PjaxAdapter {
       (type === 'commit' && typeId) ||
       // Pick the commit ID or branch name from the DOM
       // Note: we can't use URL as it would not work with branches with slashes, e.g. features/hotfix-1
-      ($('.overall-summary .numbers-summary .commits a').attr('href') || '').split('/').slice(-1)[0] ||
+      ($('.overall-summary .numbers-summary .commits a').attr('href') || '')
+        .replace(`/${username}/${reponame}/commits/`, '') ||
```

Comment:
Maybe we should move this line after line 175 (`branchFromSummary ||`).

Reply:
That's a good idea (and fixes the issue by itself).

Reply:
Yeah, this is the fallback. The other one is more robust. Thus still need this line here though in case DOM changes.